.github/workflows/ci.yml

```diff
@@ -0,0 +1,327 @@
+        if: steps.metadata.outputs.module == 'java'
+
+      ##
+      ## Node
+      ##
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '${{ steps.metadata.outputs.version }}'
+        if: steps.metadata.outputs.module == 'node'
+
+      - name: Install node-gyp
+        run: |
+          npm install -g node-gyp
+        if: steps.metadata.outputs.module == 'node'
+
+      - name: Configure node
+        run: |
+          ./configure nodejs
+        if: steps.metadata.outputs.module == 'node'
+
+      - name: Make node
+        run: |
+          make node-local-install DESTDIR=node
+        if: steps.metadata.outputs.module == 'node'
+
+      ##
+      ## Perl
+      ##
+
+      # Uses default Actions VM Perl
+      # https://github.com/actions/runner-images#available-images
+
+      - name: Install libperl-dev
+        run: |
+          sudo apt-get install libperl-dev
+        if: steps.metadata.outputs.module == 'perl'
+
+      - name: Configure perl
+        run: |
+          ./configure perl
+        if: steps.metadata.outputs.module == 'perl'
+
+      - name: Make perl
+        run: |
+          make perl
+        if: steps.metadata.outputs.module == 'perl'
+
+      ##
+      ## PHP
+      ##
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '${{ steps.metadata.outputs.version }}'
+          extensions: none
+        env:
+          update: true
+        if: steps.metadata.outputs.module == 'php'
+
+      - name: Configure php
+        run: |
+          ./configure php
+        if: steps.metadata.outputs.module == 'php'
+
+      - name: Make php
+        run: |
+          make php
+        if: steps.metadata.outputs.module == 'php'
+
+      ##
+      ## Python 3
+      ##
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '${{ steps.metadata.outputs.version }}'
+        if: steps.metadata.outputs.module == 'python'
+
+      - name: Configure python3
+        run: |
+          ./configure python --config=python3-config
+        if: steps.metadata.outputs.module == 'python'
+
+      - name: Make python3
+        run: |
+          make python3
+        if: steps.metadata.outputs.module == 'python'
+
+      ##
+      ## Ruby
+      ##
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '${{ steps.metadata.outputs.version }}'
+        if: steps.metadata.outputs.module == 'ruby'
+
+      - name: Install rack
+        run: |
+          gem install rack
+        if: steps.metadata.outputs.module == 'ruby'
+
+      - name: Configure ruby
+        run: |
+          ./configure ruby
+        if: steps.metadata.outputs.module == 'ruby'
+
+      - name: Make ruby
+        run: |
+          make ruby
+        if: steps.metadata.outputs.module == 'ruby'
+
+      ##
+      ## Wasm
+      ##
+
+      - name: Make wasmtime
+        run: |
+          make -C pkg/contrib .wasmtime
+        if: steps.metadata.outputs.module == 'wasm'
```

Comment:
What version of wasmtime does this use?

Reply:
Looks like we're at Wasmtime 11 in `pkgs/contrib`